kp_service_test/src/test/resources/templates/content/v3/hierarchy/update/testUpdateHierarchyWith1Unit1CollectionValidRequest/response.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "content.hierarchy.update",
+  "ver": "3.0",
+  "ts": "@ignore@",
+  "params": {
+    "resmsgid": "@ignore@",
+    "msgid": null,
+    "err": null,
+    "status": "successful",
+    "errmsg": null
+  },
+  "responseCode": "OK",
+  "result": {
+    "content_id": "@ignore@",
```

Comment:
please use at least "@startsWith('KP_FT_')@" if you can't assert on exact value.